tests/resources/mocks/data_provider_mock.py

```diff
@@ -57,6 +57,10 @@ def compute_job_status(*args, **kwargs):
     def compute_job_result(*args, **kwargs):
         return True
 
+    @staticmethod
+    def compute_job_result_file(*args, **kwargs):
+        return True
```

Comment:
Since the return value of `DataServiceProvider.compute_job_result_file()` is `Dict[str, Any]` shouldn't this `Mock` return something of the same type?

Reply:
I think all the methods are just dummies in place. We should change all of them, if that is the case. Maybe we can create a ticket for this? I think it's not strictly needed, but let's discuss this in its own.

Reply:
Fair enough.